src/applications/vre/28-8832/wizard/pages/Start.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ErrorableRadioButtons from '@department-of-veterans-affairs/formation-react/ErrorableRadioButtons';
+import { pageNames } from './pageList';
+
+const options = [
+  {
+    value: pageNames.isVeteran,
+    label: 'Veteran',
+  },
+  {
+    value: pageNames.isServiceMember,
+    label: 'Service member',
+  },
+  {
+    value: pageNames.VAEducationBenefits,
+    label: 'Dependent of a Veteran or service member',
+  },
+  {
+    value: pageNames.ineligibleNotice,
+    label: 'None of these',
+  },
+];
+
+const StartPage = ({ setPageState, state = {} }) => {
+  const handleValueChange = ({ value }) => {
+    switch (value) {
+      case 'isVeteran':
+      case 'isServiceMember':
+        setPageState({ selected: value }, 'isVeteranOrServiceMember');
```

Comment:
These options follow essentially the same wizard path, so they need to be handled in the same manner.